kumascript/macros/FirefoxSidebar.ejs

```diff
@@ -35,12 +35,12 @@ const text = mdn.localStringMap({
   },
   "fr": {
     "Firefox_releases": "Versions de Firefox",
-      "Firefox_release_notes_developer": "Firefox release notes for developers",
-      "Experimental_features_firefox": "Experimental features in Firefox",
+      "Firefox_release_notes_developer": "Notes de version Firefox pour les développeurs",
```

Comment:
```suggestion
      "Firefox_release_notes_developer": "Notes de version Firefox pour le développement",
```
Formulation épicène et cohérence avec https://transvision.mozfr.org/?recherche=for+developers&repo=gecko_strings&sourcelocale=en-US&locale=fr&search_type=strings